.github/CHANGELOG.md

```diff
@@ -497,16 +526,16 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
   Temporary backward compatibility has been added to support the use of `_stepsize` for one
   release cycle. `update_stepsize` method is deprecated.
   [(#1625)](https://github.com/PennyLaneAI/pennylane/pull/1625)
-  
+
 
 <h3>Bug fixes</h3>
 
 * Fix bug with shot vectors and `Device` base class.
```

Comment:
Very _very_ minor, but it might be nice to standardize the 'fix bug' vs 'fixed a bug' :)